definitions/divorce/json/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -82,5 +82,33 @@
     "CaseEventID": "amendPetitionForRefusalSol",
     "UserRole": "caseworker-divorce-courtadmin-la",
     "CRUD": "R"
+  },
+  {
+    "LiveFrom": "03/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "allowPetitionerToAmend",
+    "UserRole": "caseworker-divorce-courtadmin_beta",
+    "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "03/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "allowPetitionerToAmend",
+    "UserRole": "caseworker-divorce-solicitor",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "03/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "allowPetitionerToAmend",
+    "UserRole": "caseworker-divorce-courtadmin-la",
+    "CRUD": "R"
+  },
+  {
+    "LiveFrom": "03/07/2020",
+    "CaseTypeID": "DIVORCE",
+    "CaseEventID": "allowPetitionerToAmend",
+    "UserRole": "caseworker-divorce-superuser",
+    "CRUD": "CRU"
```

Comment:
Might need to confirm with Jeremy if super users should have right to run this event. They probably should I guess.

Reply:
<img width="421" alt="Screenshot 2020-07-06 at 11 34 22" src="https://user-images.githubusercontent.com/30896346/86584340-aa2ac100-bf7c-11ea-911b-7870fb647460.png">

Reply:
<img width="421" alt="Screenshot 2020-07-06 at 11 34 22" src="https://user-images.githubusercontent.com/30896346/86584401-c3337200-bf7c-11ea-863a-f7c7cf4c3a24.png">
Confirmed

Reply:
cool aha